src/lib.rs

```diff
@@ -13,12 +18,48 @@ use std::sync::{ Arc, Mutex, MutexGuard };
 use std::sync::mpsc;
 
 #[cfg(windows)]
-fn get_default_device() -> Result<&'static str> {
-    "\\\\.\\COM6"
+fn get_default_device() -> Result<String> {
+    "\\\\.\\COM6".to_owned()
+}
+
+fn is_usb_zwave_device(port: &ListPortInfo) -> bool {
+    let default_usb_devices = [
+        // VID     PID
+        //-----   -----
+        (0x0658, 0x0200),   // Aeotech Z-Stick Gen-5
+    ];
+
+    // Is it one of the vid/pids in the table?
+    if let UsbPort(ref info) = port.port_type {
+        default_usb_devices.iter().find(|entry| entry.0 == info.vid && entry.1 == info.pid).is_some()
```

Comment:
I think you can simply use `default_usb_devices.contains((info.vid, info.pid))` because `(T, U)` implements PartialEq if T and U do.